docs/ProductSummaryImage.md

```diff
@@ -44,78 +44,78 @@
+|:--------------------:|:--------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------:|
+|      `criteria`      |  `enum`  |                                                                                                             Criteria that should determine the hover image based on the product images in the Admin Catalog. Possible values are: `label` (the hover image will be the one that matches the `label` value) and `index` (the hover image will be the one with the same `index` value).                                                                                                              |    `label`    |
+|       `label`        | `string` |                                                                                                                                           Text string to match the desired image's `label` value. No image will be displayed during the hover if no match is found. *Caution*: This prop should only be used when the `criteria` prop is set to `label`.                                                                                                                                           |  `undefined`  |
+| `labelMatchCriteria` |  `enum`  | Criteria for searching the image `label` value. It determines whether the image label should match exactly as provided or if it just needs to contain the substring anywhere in the image `label`. Possible values are: `exact` (finds the image that matches exactly the string filled in the `label` field) and `contains` (finds the first image that includes the substring filled in the `label` field). *Caution*: This prop should only be used when the `criteria` prop is set to `label`. |    `exact`    |
+|       `index`        | `number` |                                                                                                                                               Index number to match with the desired images. No image will be displayed during the hover if no match is found. *Caution*: This prop should only be used when the `criteria` prop is set to `index`.                                                                                                                                                |  `undefined`  |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| - | - | - | - |
| `criteria` | `enum` | Criteria that should determine the hover image based on the product images in the Admin Catalog. Possible values are: `label` (the hover image will be the one that matches the `label` value) and `index` (the hover image will be the one with the same `index` value). | `label` |
| `label` | `string` | Text string to match the desired image's `label` value. No image will be displayed during the hover if no match is found. *Caution*: This prop should only be used when the `criteria` prop is set to `label`. |  `undefined`  |
| `labelMatchCriteria` |  `enum`  | Criteria for searching the image `label` value. It determines whether the image label should match exactly as provided or if it just needs to contain the substring anywhere in the image `label`. Possible values are: `exact` (finds the image that matches exactly the string filled in the `label` field) and `contains` (finds the first image that includes the substring filled in the `label` field). *Caution*: This prop should only be used when the `criteria` prop is set to `label`. | `exact` |
| `index` | `number` | Index number to match with the desired images. No image will be displayed during the hover if no match is found. *Caution*: This prop should only be used when the `criteria` prop is set to `index`. | `undefined` |
```

Reply:
@GeorgeLimaDev please do not edit original table formatting, especially if it's already in compact markdown. It's much easier for tech writers to edit and review compact markdown tables without many blank spaces.